functions/__pure_prompt_beginning.fish

```diff
@@ -1,4 +1,4 @@
-function __pure_new_line
+function __pure_prompt_beginning
```

Comment:
should be either:
* `__pure_prompt_begin` /`__pure_prompt_end` 
* `__pure_prompt_beginning` / `__pure_prompt_ending`